types/matrix-js-sdk/tslint.json

```diff
@@ -0,0 +1,26 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove all the rules from here and fix errors.. New packages cannot disable any lint rule

Reply:
Removed in the latest commit.